precompile/registry/registry.go

```diff
@@ -16,6 +17,7 @@ import (
 // Note: it is important that none of these addresses conflict with each other or any other precompiles
 // in /coreth/contracts/contracts/**.
 
-// WarpMessengerAddress = common.HexToAddress("0x0200000000000000000000000000000000000005")
+// WarpMessengerAddress  = common.HexToAddress("0x0200000000000000000000000000000000000005")
+// P256VerifyAddress     = common.HexToAddress("0x0000000000000000000000000000000000000100")
```

Comment:
Updated address in line with https://github.com/avalanche-foundation/ACPs/pull/222

Reply:
again this is only for a reference to stateful precompile addresses. we should not include this one here.